pkg/ring/model.go

```diff
@@ -431,6 +431,74 @@ func (d *Desc) getTokensByZone() map[string][]TokenDesc {
 	return zones
 }
 
+type CompareResult int
+
+const (
+	Equal                   CompareResult = iota // Both rings contain same instances, tokens, states and timestamps.
+	EqualInstancesAndTokens                      // Both rings contain the same instances and tokens, but states and timestamps differ.
+	Different                                    // Rings have different set of instances, or their tokens don't match.
```

Comment:
It's not just tokens but other info too, like zone.

```suggestion
	Different                                    // Rings have different set of instances, or their information don't match.
```